app/bundles/LeadBundle/EventListener/PointSubscriber.php

```diff
@@ -66,13 +54,21 @@ public function onTriggerBuild(TriggerBuilderEvent $event)
             ]
         );
 
+        $stages =$this->stageModel->getUserStages();
+
+        $choices['remove existing stage'] = 0;
```

Comment:
is this displayed to the user? then it needs to be translatable. sorry :) 

EN: Remove existing stage
DE: Aktuelle Stage entfernen

Reply:
maybe thats a personal choice, but I would put the string in the value
0 => Remove existing stage
...

Or I guess the best one would be to create an object:
[{
label: Remove existing stage
id: 0
}]

At least that is how I would do it in TypeScript.

Reply:
This array is added as choises to a symfony form. It should have the structure key => value